app/controllers/api/v4/states_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V4::StatesController < APIController
+  include Api::V3::PublicApi
+
+  def sync_to_user
+    state_names = Region.facility_regions
+      .joins("inner join regions states ON states.path @> regions.path and states.region_type = 'state'")
+      .distinct("states.name")
+      .pluck("states.name")
```

Comment:
What's the difference between this and `Region.state_regions`?

Reply:
`Region.state_regions` can have states that don't have any facility under them. To get states that have at least one facility we could:
- Start from `Region.state_regions` and join a distinct facility on each state, filter states that have a facility
- Start from `Region.facility_regions` and get their states

The latter seemed simpler to write.

Reply:
Ah I see, the state regions are _every_ state in the country from the canonical list. Neat neat.